src/modules/universal-discovery/services/universal.discovery.service.js

```diff
@@ -59,6 +59,82 @@ export const loadContentInfo = ({token, siteaccess}, contentId, callback) => {
         .catch(error => console.log('error:load:content:info', error));
 };
 
+/**
+ * Loads location
+ *
+ * @function loadLocation
+ * @param {Object} params params hash containing REST config: token and siteaccess properties; locationId and offset
+ * @param {Function} callback
+ */
+export const loadLocation = ({
+        token,
+        siteaccess,
+        locationId,
+        limit = QUERY_LIMIT,
+        offset = 0
+    }, callback) => {
+    const body = JSON.stringify({
+        ViewInput: {
+            identifier: `udw-location-by-id-${locationId}`,
+            public: false,
+            LocationQuery: {
+                Criteria: {},
+                FacetBuilders: {},
+                SortClauses: {},
+                Filter: {LocationIdCriterion: locationId},
+                limit,
+                offset
+            }
+        }
+    });
+    const request = new Request(ENDPOINT_CREATE_VIEW, {
+        method: 'POST',
+        headers: Object.assign({}, HEADERS_CREATE_VIEW, {
+            'X-Siteaccess': siteaccess,
+            'X-CSRF-Token': token
+        }),
+        body,
+        mode: 'cors',
+        credentials: 'same-origin'
+    });
+
+    fetch(request)
+        .then(handleRequestResponse)
+        .then(json => callback(json))
+        .catch(error => console.log('error:load:location:by:id', error));
+};
+
+/**
+ * Checks if user has permission to create content
+ *
+ * @function checkCreatePermission
+ * @param {Object} params params hash containing REST config: token, siteaccess, contentTypeIdentifier, languageCode, locationId
+ * @param {Function} callback
+ */
+export const checkCreatePermission = ({
+        token,
+        contentTypeIdentifier,
+        languageCode,
+        locationId
+    }, callback) => {
+    const endpoint = window.Routing.generate('ezplatform.content.create.on_the_fly.has_access', {
```

Comment:
There's no need to duplicate the names: `window.Routing.generate('ezplatform.content.create.on_the_fly.has_access', { languageCode, contentTypeIdentifier, locationId })`